_episodes/twir/2022-06-06-twir-444.md

```diff
@@ -0,0 +1,202 @@
+
+- Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+- Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+### Timestamps & referenced resources
+
+<!--
+In this section, leave timestamped notes of the form:
+
+ - [@HH:MM:SS] - Topic at first timestamp
+ - [@HH:MM:SS] - Topic at second timestamp
+     - A link to additional material discussed during the preceding topic
+
+-->
+
+#### [@00:00] Welcome
+
+- [@00:10] - Introduction
+
+- [@00:50] - Agenda
+
+- [@01:23] - [Quote of the week](https://lobste.rs/s/wiavtb/rust_critical_retrospective#c_jkfhpb)
+  > This is the difference in approaches of the two languages. In C++ if the
+  > code is vulnerable, the blame is on the programmer. In Rust if the code is
+  > vulnerable, Rust considers it a failure of the language, and takes
+  > responsibility to stop even “bad” programmers from writing vulnerable code.
+  > I can’t stress enough how awesome it is that I can be a careless fool, and
+  > still write perfectly robust highly multi-threaded code that never crashes.
+  - [@03:09] Allen: Rust is both good and bad at marketing
+
+- [@03:30] - [Crate of the week](https://docs.rs/rustdoc-types)
+  - [@04:15] - Tim and Sean discuss parsing in [episode
+    2022-05-26](https://rustacean-station.org/episode/071-twir-442/) at 47:10
+
+#### [@05:10] Official Notices
+
+- [@05:22] - [Announcing Rust 1.61.0](https://blog.rust-lang.org/2022/05/19/Rust-1.61.0.html)
+  - Custom exit codes from main
+    - [Note from Tim: I say "termination crate", but should have said "Termination trait".]
+  - More capabilities for const fn
+    - "Basic" handling of fn pointers
+    - Add trait bounds to a const fn
+    - dyn trait and impl Trait support
+  - Stdio handles can be locked directly
+  - Several stabilized APIs
+
+#### [@08:07] Highlights
+
+- [@08:27] - [Developer survey: JavaScript and Python reign, but Rust is rising](https://www.infoworld.com/article/3661248/developer-survey-javascript-and-python-reign-but-rust-is-rising.html)
+  - [@09:09] - Sean: "Rust adoption has nearly quadrupled in the last two years, going from
+   600k developers in Q1 2022 to 2.2m in Q1 2022."
+
+- [@13:00] - [Redust](https://github.com/appellation/redust) by [Will Nelson](https://wnelson.dev/)
+  - [@13:50] Allen: I think the comments are actually more interesting. They
+    are starting to point to something really&mdash;I don't know whether it's
+    good or bad for the community&mdash;where, you know, people start rolling
+    their own crates instead of, say, doing stuff upstream. It kind of goes back
+    to what [Tim was complaining about before](https://rustacean-station.org/episode/069-twir-441/) [Easy Mode for
+    Rust, discussed on This Week in Rust - Issue 441]&mdash;well, lightly
```

Comment:
The "Easy Mode" link here seems broken.

Reply:
Oh, I used square brackets here because it's meta-discussion. It's not part of the quote itself.

Reply:
I wonder how many questions we'll get about people going "the link is broken" :p

Reply:
I wonder how many people will read that far down the page 😅